packages/wonder-blocks-core/components/view.js

```diff
@@ -59,17 +59,22 @@ export default class View extends React.Component<Props> {
 
     render() {
         const {testId, tag, ...restProps} = this.props;
+        const props = {
+            ...restProps,
+            "data-test-id": testId,
+        };
```

Comment:
Flow worries that spreading an object with an index will overwrite props that come before the spread which is a valid concern.